tensorflow/core/distributed_runtime/worker.cc

```diff
@@ -25,6 +25,12 @@ limitations under the License.
 
 namespace tensorflow {
 
+// Find worker session by session handle. It's will be not null.
```

Comment:
Replace "It's will be not null." with "REQUIRES: `request` is not null. The return value is guaranteed not to be null."

Reply:
OK, I fixed it.